atomic_reactor/utils/pnc.py

```diff
@@ -18,9 +19,43 @@ def __init__(self, pnc_map, session=None):
             self.session = get_retrying_requests_session()
         else:
             self.session = session
-        self.base_api_url = pnc_map['base_api_url']
-        # using urljoin here causes the API path in the base_api_url to be removed
-        self.get_scm_archive_request_url = self.base_api_url + '/' + pnc_map['get_scm_archive_path']
+
+        self.pnc_map = pnc_map
+        self.base_api_url = self.pnc_map['base_api_url']
+        self._get_artifact_url = None
+        self._get_scm_archive_request_url = None
+
+    @property
+    def get_artifact_url(self):
```

Comment:
if this is a property, then is should be called just `artifact_url`

Reply:
updated.